snippet-extractor-output/snippets.yaml

```diff
@@ -5129,6 +5129,82 @@
         textbox.textFrame.deleteText();
         await context.sync();
     });
+'Excel.WebImageCellValue#address:member':
```

Comment:
Is there value in having one of these two samples on the same page map to the `openImage` function instead? That way, you're showing both get and set functionality.